src/lib/components/PriceTag.svelte

```diff
@@ -1,6 +1,7 @@
 <script lang="ts">
 	import {
 		FRACTION_DIGITS_PER_CURRENCY,
+		getStoragePrecision,
```

Comment:
Why does the price tag display performs price computation?

This is a display component, it shouldn't care about how the price is stored.